habitat-lab/habitat/tasks/rearrange/rearrange_sensors.py

```diff
@@ -1656,6 +1664,178 @@ def get_observation(self, observations, *args, **kwargs):
                 self.pcl_o3d_list = []
                 self._debug_save_counter += 1
 
-    
+
         return rgb_obs
-        
\ No newline at end of file
+
+
+@registry.register_sensor
+class ArmWorkspaceRGBArmSensor(ArmWorkspaceRGBSensor, UsesArticulatedAgentInterface, Sensor):
+    """ Sensor to visualize the reachable workspace of an articulated arm """
+    cls_uuid: str = "arm_workspace_rgb_arm"
```

Comment:
Can we add a TODO line in docstring to mark this function as buggy code for now? And also hint the issue could be in arm camera transformation in habitat-sim not consistent with that in habitat-lab.

Reply:
Sure.